packages/fetch-http-handler/src/stream-collector.ts

```diff
@@ -45,7 +45,10 @@ function readToBase64(blob: Blob): Promise<string> {
         return reject(new Error("Reader aborted too early"));
       }
       const result = reader.result as string;
-      const dataOffset = result.indexOf(",") + 1;
```

Comment:
need to add `null` check as null.indexOf will throw
`Cannot read property 'indexOf' of null`

Reply:
fixed in [commit](https://github.com/aws/aws-sdk-js-v3/pull/1199/commits/1e5fa454a42f0219eca536d2356f7191804cabb0)